subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -708,7 +708,8 @@ void ull_sync_setup(struct ll_scan_set *scan, struct ll_scan_aux_set *aux,
 	sync->ull.ticks_slot =
 		HAL_TICKER_US_TO_TICKS(EVENT_OVERHEAD_START_US +
 				       ready_delay_us +
-				       PDU_AC_MAX_US(0U, lll->phy) +
+				       PDU_AC_MAX_US(PDU_AC_EXT_PAYLOAD_SIZE_MAX,
```

Comment:
It looks that this change reverts change done in this PR: 779d523c487a55267113177b617ffef79df90963.

Reply:
Yes, the use of ticker_yield_abs does not need minimal reservation. I would like to know the commit with minimal reservation for history of the issue being addressed.